.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "recommendations": [
```

Comment:
Please align the extensions (and settings) with the `.devcontainer` file:

https://github.com/Together-Java/TJ-Bot/blob/develop/.devcontainer/devcontainer.json#L17-L32

So that both setups use the same stuff.